pkg/controller/flink/job_manager_controller_test.go

```diff
@@ -3,6 +3,8 @@ package flink
 import (
 	"testing"
 
+	v1beta12 "github.com/lyft/flinkk8soperator/pkg/apis/app/v1beta1"
```

Comment:
nit, would be good to give this a more descriptive name

Reply:
Changed to `flinkapp`